packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -55,6 +63,14 @@ export class TableColumnNumberText extends TableColumnTextBase {
     })
     public decimalMaximumDigits?: number;
 
+    /** @internal */
```

Comment:
I'm still wrapping my head around this PR so might have missed something, but are there tests for the logic in this file? I was expecting changes in `table-column-number-text.spec.ts` to validate things like adding/removing/changing attributes of unit elementsand the new case for left/right default alignment.

Reply:
Good catch. I missed adding some tests there. I've added cases for:
- adding/removing a unit element
- setting an attribute of a unit element
- configuring a unit in default formatting mode
- a few new alignment scenarios